omnibus/package-scripts/chef-server/postinst

```diff
@@ -52,6 +52,10 @@ if [ -e /etc/opscode/chef-server-running.json ]; then
   echo
   echo "For detailed upgrade instructions please see:"
   echo "https://docs.chef.io/server/upgrades/"
+else
+  echo -e "\033[1mThank you for installing Chef Infra Server!\033[0m"
+  echo
+  echo "For more information on getting started see https://docs.chef.io/server/"
```

Comment:
Should it point to the install page instead? https://docs.chef.io/server/install_server/